.jvmopts

```diff
@@ -1,2 +1,2 @@
--Xmx1024m
+-Xmx2048m
```

Comment:
When I pulled down this PR and opened it in IntelliJ, my laptop grinded to an absolute halt. Had to reboot. I'm no expert in Java memory allocation, but I once tweaked the default IntelliJ memory parameters, causing IntelliJ to several times a day hog every single byte available on my machine. Do you think this option change might behave badly in some situations? 

It might be that my laptop crashed for some other reason as well.